tools/add_triage_signature.py

```diff
@@ -345,6 +345,33 @@ def _get_hostname(host):
         inventory = json.loads(host["inventory"])
         return inventory["hostname"]
 
+    @staticmethod
+    def _generate_hosts_summary(hosts):
+        host_summary = []
+        for host in hosts:
+            # Add host status summary
+            if host["role"] not in ["master", "boostrap"]:
+                continue
+            if host["progress"] == "Done" or host["status"] != "error":
+                continue
+
+            host_comment = {
+                "Waiting for bootkube": "Bootkube.service never started",
+                "Rebooting": "Node never pulled Ignition",
+                "Configuring": "Node pulled Ignition, but never started kubelet",
+                "Waiting for control plane": "Masters never formed 2-node cluster",
+                "Waiting for controller": "Assisted installer controller pod never started",
+                "Writing image to disk": "Image failed to be written on disk",
```

Comment:
This is a bit misleading because sometimes a stage ends prematurely when the cluster goes into error. But a cluster can go into error due to circumstances outside those happening on the node with the stage.

e.g. if workers are in the middle of writing the image to disk, and suddenly the bootstrap node unexpectedly disconnects from the service, the AI cluster will go into the error state, and as a result of that transition all hosts within the cluster will also move to the error state. But their "stage" (stages are "sub-states" of the Installing state) will remain the same and appear in that table.  

```suggestion
                "Writing image to disk": "Image probably failed to be written on disk",
```